.travis.yml

```diff
@@ -26,5 +26,7 @@ install:
   - alias md5=md5sum
   - pip install -e . --no-deps
 script:
-  - flake8 --ignore=E731 redbiom 
+  - flake8 --ignore=E731 redbiom
+  - redbiom summarize contexts  # will return a nonzero exit status if it cannot communicate with the default host
```

Comment:
I don't get this, you are testing if it can communicate (which it will fail and stop execution) and then change REDBIOM_HOST? Would you mind explaining why is this required?